### PR TITLE
Adding new Systems

### PR DESCRIPTION
**Pull request by sourabhdalvi**
- Adding Full RTS-GMLC system
- Adding Modified RTS-GMLC system for day-ahead and real-time
- Fixes issue #7

### CODE REVIEW

File: data/data_5bus_pu.jl [hunk 1]
```diff
@@ -913,6 +913,11 @@ ORDC_cost_ts = [
     TimeSeries.TimeArray(DayAhead + Day(1), repeat([ORDC_cost], 24)),
 ]
 
+hybrid_cost_ts = [
```
Comment (jd-lara) [inline data/data_5bus_pu.jl:916]: Lets try to make this values reasonable or add a comment about why this value

Comment (sourabhdalvi) [reply data/data_5bus_pu.jl:916]: I'm not sure what value is sensible, I did change it to my best guess and also added a comment

File: src/system_descriptor_data.jl [hunk 2]
```diff
@@ -216,6 +216,30 @@ const SYSTEM_CATELOG = [
         raw_data = joinpath(PACKAGE_DIR, "data", "RTS_GMLC"),
         build_function = build_test_RTS_GMLC_sys,
     ),
+    SystemDescriptor(
+        name = "RTS_GMLC_sys",
+        description = "RTS-GMLC Full system from git repo",
```
Comment (claytonpbarrows) [inline src/system_descriptor_data.jl:221]: is this an hourly or 5-minute system?

File: src/system_descriptor_data.jl [hunk 3]
```diff
@@ -216,6 +216,30 @@ const SYSTEM_CATELOG = [
...
+        category = PSITestSystems,
+        raw_data = joinpath(PACKAGE_DIR, "data", "RTS-GMLC-master"),
+        build_function = build_RTS_GMLC_sys,
+        download_function = download_RTS,
+    ),
+    SystemDescriptor(
+        name = "modified_RTS_GMLC_DA_sys",
+        description = "Modified RTS-GMLC Full system for day-ahead simulations",
```
Comment (claytonpbarrows) [inline src/system_descriptor_data.jl:229]: add some mention of what the modifications are... maybe "RTS-GMLC Full system for day-ahead simulations with modifications to reserve definitions to improve feasibility"

File: src/system_descriptor_data.jl [hunk 4]
```diff
@@ -216,6 +216,30 @@ const SYSTEM_CATELOG = [
...
+        category = PSITestSystems,
+        raw_data = joinpath(PACKAGE_DIR, "data", "RTS-GMLC-master"),
+        build_function = build_modified_RTS_GMLC_DA_sys,
+        download_function = download_RTS,
+    ),
+    SystemDescriptor(
+        name = "modified_RTS_GMLC_RT_sys",
+        description = "Modified RTS-GMLC Full system for real-time simulations",
```
Comment (claytonpbarrows) [inline src/system_descriptor_data.jl:237]: same here

**Review by claytonpbarrows — APPROVED** [review]

File: src/system_descriptor_data.jl (hunk 3 above)
Comment (sourabhdalvi) [reply src/system_descriptor_data.jl:229]: yup will do that

**Pull request merged**